resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
We are more flexible in NGDA than we are with iJento, the main reason we have to identify pages in iJento is the ability to tell if they are "counted content". iJento could only do this on a page.

In WebApp you might only send one page for the whole app and then send events OR we might might model pages again. Partly it's a technical decision, partly it's a people-who-are-reporting-on-this-stuff decision.

In NGDA we are flexible to handle both, we are also thinking about a world without pages in the far future, but thats outside the scope of the NGDA project.